linode/rdns/framework_resource.go

```diff
@@ -41,11 +42,34 @@ func (r *Resource) Create(
 
 	plan.ID = plan.Address
 
+	ip, err := client.GetIPAddress(ctx, plan.Address.ValueString())
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Failed to get the ip address associated with this RDNS",
+			err.Error(),
+		)
+		return
+	}
+
+	defaultRdns := strings.Replace(
+		plan.Address.ValueString(),
+		".",
+		"-",
+		-1,
+	) + ".ip.linodeusercontent.com"
+
+	if ip.RDNS != defaultRdns {
+		resp.Diagnostics.AddWarning(
+			"Pre-exist RDNS configuration",
```

Comment:
Tiny language nitpick:
```suggestion
			"Pre-modified RDNS Address",
```